src/clib/lib/include/ert/enkf/model_config.hpp

```diff
@@ -41,6 +39,13 @@
 #include <ert/enkf/fs_types.hpp>
 #include <ert/enkf/time_map.hpp>
 
+typedef enum {
+    /** ecl_sum_get_well_var( "WWCT" );  */
+    REFCASE_SIMULATED = 1,
+    /** ecl_sum_get_well_var( "WWCTH" ); */
```

Comment:
Perhaps pedandtic here, but does it work to use a c++17 style enum definition? 
```cpp
using history_source_type = enum {
    /** ecl_sum_get_well_var( "WWCT" );  */
    REFCASE_SIMULATED = 1,
    /** ecl_sum_get_well_var( "WWCTH" ); */
    REFCASE_HISTORY = 2,
};

```

Reply:
Should be fine, I just moved it as it was, but makes sense to update :+1: